### ISSUE TITLE
Javascript issue w/quick reply

### ISSUE THREAD

**Issue by GoogleCodeExporter** [VBGAMER45/prettyurls#87]
```
What steps will reproduce the problem?
1. Enable pretty urls for topic
2. Enable quick reply
3. Click on the collapse button

What is the expected output? What do you see instead?
Quick reply box.  404 error

What version of the mod, and what version of SMF are you using?
1.0RC, 1.1.9

What other mods do you have installed?
Advanced quick reply.

Please provide any additional information below.


```

Original issue reported on code.google.com by `elochaib...@gmail.com` on 7 Jul 2009 at 11:28

**Comment by GoogleCodeExporter** [comment]
```
Are you using the original or the patched package of 1.0RC?
```

Original comment by `curiousdannii` on 9 Jul 2009 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
Most likely was fixed in the 1.0RC patch.
```

Original comment by `curiousdannii` on 21 Jul 2009 at 5:18
- Changed state: **Fixed**
